src/Containers/containerize/containerize.csproj

```diff
@@ -17,4 +17,11 @@
     <ProjectReference Include="..\Microsoft.NET.Build.Containers\Microsoft.NET.Build.Containers.csproj" />
   </ItemGroup>
 
+  <!-- Copy to TestExecutionDirectory for using in integration tests. -->
+  <Target Name="CopyDllToTestExecutionDirectory" AfterTargets="Build">
```

Comment:
Need to copy self-contained build output to artifact temp folder and then copy to Helix. 
Due to my poor knowledge on msbuild targets, this didn't work. Need to improve.